charts/studio/templates/ingress-studio-api.yaml

```diff
@@ -0,0 +1,60 @@
+{{- if .Values.global.ingress.enabled -}}
+{{- if and .Values.global.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
+  {{- if not (hasKey .Values.global.ingress.annotations "kubernetes.io/ingress.class") }}
+  {{- $_ := set .Values.global.ingress.annotations "kubernetes.io/ingress.class" .Values.global.ingress.className}}
+  {{- end }}
+{{- end }}
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1
+{{- else if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
+kind: Ingress
+metadata:
+  name: app-api
```

Comment:
```suggestion
  name: {{ .Release.Name }}-studio-api
```

Reply:
We will have to propagate `{{ .Release.Name }}` on everyresource, to allow deploying multiple releases on same namespace(it was an issue with skaffold)

Reply:
And also we need to make the naming consistent. Somwhere we are using `studio`, somewhere is `app` :D

Reply:
> And also we need to make the naming consistent. Somwhere we are using studio, somewhere is app :D

That's a good point. The `app` designation was so that we could have a distinction between the core Studio components and its dependencies